docs-svelte-kit/src/.eslintrc.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
     "node/no-unsupported-features/es-syntax": "off",
     "require-jsdoc": "off",
     "node/no-missing-import": "off",
+    "@ota-meshi/missing-module-for-config": "off",
+    "prettier/prettier": "off",
```

Comment:
Svelte components in `docs-svelte-kit` has  below 2 errors but I don't know how to solve these.
So for now, I disabled them but this is terrible, so could you please teach me how to solve them?

<img width="994" alt="image" src="https://user-images.githubusercontent.com/19153718/198863565-7f15d05f-a433-42f3-ac62-34a5488b3cf0.png">


<img width="519" alt="image" src="https://user-images.githubusercontent.com/19153718/198863569-fa13877e-a1df-4c08-a9f0-a0dd4f9b6b9f.png">

Reply:
Maybe we need to update eslint-plugin-svelte to >=v2.9.
I will open it PR.

Reply:
Could you merge the main branch, install it, and try it? #285

Reply:
Oh Thank you.
Now it's working fine and I reverted it.

https://github.com/ota-meshi/eslint-plugin-svelte/pull/281/commits/d0d21e75c323bb843bd32227a9abcf1f0bcd798a